### DISCUSSION TITLE
Https

### DISCUSSION THREAD

**Discussion by Abreu70** [community/community#82226] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

No puedo activar la placa de https. Tengo dominio propio generado con goddady.

**Comment by Akash1134** [comment, marked as answer]
Hi USERNAME,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this. one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
